packages/console/components/duckdb.ts

```diff
@@ -0,0 +1,53 @@
+import { ConsoleLogger, AsyncDuckDB, AsyncDuckDBConnection } from '@duckdb/duckdb-wasm';
+import duckdb_wasm from '@duckdb/duckdb-wasm/dist/duckdb-mvp.wasm?url';
+import duckdb_worker from '@duckdb/duckdb-wasm/dist/duckdb-browser-mvp.worker.js?worker';
+import { browser } from '$app/environment';
+
+let db: AsyncDuckDB;
```

Comment:
How does this behave if there are multiple instances of `Console` on the page at a time? 
Is this a use-case we want to support? Are there performance implications with having multiple? 
The worker behavior is probably worth checking out in this case.

Reply:
It should work fine as long as none of the tables are duplicated names (which is actually probably something I should change; right now everything goes under `data`)

Reply:
Hmm looks like it (duckdb-wasm in general) is actually pretty unstable with multiple instances on the same page. It works about half the time and throws an error about the table not existing other times